### PR TITLE
testing distributed sharginUserInfo

### PR DESCRIPTION
**Pull request by ymatsuda**
@raymondng @eishay 
- distributed version of sharingUserInfo
- enabled only for me for testing

### CODE REVIEW

**Comment by eishay** [comment]
lgtm, though usually we use Experiments for this.

File: kifi-backend/modules/common/app/com/keepit/search/SearchServiceClient.scala [hunk 1]
```diff
@@ -148,6 +148,10 @@ class SearchServiceClientImpl(
   }
 
   def sharingUserInfo(userId: Id[User], uriId: Id[NormalizedURI]): Future[SharingUserInfo] = consolidateSharingUserInfoReq((userId, uriId)) { case (userId, uriId) =>
+    if(userId.id == 7L) return distRouter.call(uriId, Search.internal.sharingUserInfo(userId), Json.toJson(Seq(uriId.id))) map { r =>
```
Comment (raymondng) [inline kifi-backend/modules/common/app/com/keepit/search/SearchServiceClient.scala:151]: Magic number! Maybe a small function (with a bit of documentation) that encapsulates this?

**Comment by raymondng** [comment]
lgtm!

**Comment by ymatsuda** [comment]
I will enable this for everyone after verification. The old code will be removed in an hour or so. Thus, no experiment is necessary. This awkward "if" statement will be gone!

**Pull request merged**